cfme/tests/infrastructure/test_host.py

```diff
@@ -312,3 +312,14 @@ def _finalize():
             actual_state, power_state
         ),
     )
+
+
+@pytest.mark.meta(automates=[1528859])
+@pytest.mark.provider([RHEVMProvider], required_fields=['hosts'],
+                      override=True, selector=ONE, scope='module')
+def test_hosts_not_displayed_several_times(appliance, provider, setup_provider):
+    host_count = navigate_to(appliance.collections.hosts, "All").paginator.items_amount
+    provider.delete(cancel=False)
+    provider.wait_for_delete()
+    provider.create()
+    assert host_count == navigate_to(appliance.collections.hosts, "All").paginator.items_amount
```

Comment:
I don't really like this repeating line `navigate_to(appliance.collections.hosts, "All").paginator.items_amount`, is it possible to get rid of this?

Reply:
well yes, by creating a new one line function :D 
my code looks clear like Plitvice lakes to me now XD